apidocs/src/show_permitted_roles.md

```diff
@@ -0,0 +1,54 @@
+## Show permitted roles [/resources/{account}/{kind}/{id}?permitted_roles=true&privilege={privilege}]
+
+### Show permitted roles [GET]
+
+Lists the roles which have the named permission on a resource.
+
+<!-- include(partials/resource_kinds.md) -->
+
+#### Example with `curl` and `jq`
+
+Suppose your organization name is "mycorp" and you want to find out which roles have `execute` privileges on the Variable `db-password`, and can thus fetch the secret:
+
+```bash
+curl -H "$(conjur authn authenticate -H)" \
+     'https://eval.conjur.org/resources/mycorp/variable/db-password?permitted_roles=true&privilege=execute' \
+     | jq .
+```
+
+---
+
+#### Request
+
+<!-- include(partials/auth_header_table.md) -->
+
+#### Response
+
+| Code | Description                             |
+|------|-----------------------------------------|
+|  200 | Permitted roles returned as a JSON list |
+|<!-- include(partials/http_401.md) -->|
+|<!-- include(partials/http_403.md) -->|
+|<!-- include(partials/http_422.md) -->|
+|  400 | The requested resource does not exist   |
```

Comment:
Shouldn't this be `404`? Also, probably want to move it up to be in numerical order w/ the rest of the errors.